plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list/payment-gateway-list.scss

```diff
@@ -78,20 +102,75 @@
 
 			.woocommerce-woopayments-payment-methods-logos {
 				flex-wrap: wrap;
+				padding: $gap-small 0 0 0; /* Top, Right, Bottom, Left */
+			}
+
+			@media screen and ( max-width: 782px ) {
+				margin-top: $gap-small;
+			}
+		}
+
+		&-buttons {
+			margin-left: auto;
+
+			@media screen and ( max-width: 782px ) {
+				order: 4;
+				grid-column: 2; /* Start below the 2nd column */
+				grid-row: 2; /* Move to the second row */
+				margin-left: 0;
+				margin-top: $gap;
+			}
+
+			&__actions {
+				display: flex;
+				gap: $gap;
+
+				.components-button {
+					height: 32px;
+				}
 			}
 		}
 
 		&-after {
+			margin-left: $gap;
+
+			@media screen and ( max-width: 782px ) {
+				order: 3;
```

Comment:
please add comment why it's 3

Reply:
I’ve added comments explaining the order numbers.